docs/src/components/Demo.tsx

```diff
@@ -50,16 +42,17 @@ export const Demo = ({
           <View overflow="auto" padding="5px">
             {children}
           </View>
+          <Divider
+            label="Props"
+            margin="20px 0 0"
+            border={`2px solid ${tokens.colors.brand.primary[80]}`}
+            style={{
+              '--amplify-components-divider-label-color':
```

Comment:
Have to use this trick to change the label color. Maybe we should consider add a `labelColor` prop on `Divider` ? Also, there is no prop way to change the `Divider` color

Reply:
+1 it would be great to be able to customize those separately!